### DISCUSSION TITLE
Case Design and Build

### DISCUSSION THREAD

**Discussion by sethvoltz** [sethvoltz/OctoPrint-DisplayPanel#32] [Show and tell]
Hello DisplayPanel community! Please use this discussion to show off your builds and talk about case design!

**Comment by bluespaniel** [comment]
To kickstart this conversation here is a pic of my implementation of the Octoprint-DisplayPanel.

I incorporated the screen and buttons into the skirt of my Voron 0 which required some 3D printed mounts for the switches that I solvent welded into the hexagonal pockets in the skirt. I had to bend the screens pins down to ensure good clearance with my printers PSU.

I think it looks pretty good.

![Image](https://joannes-kinky-forum.com/download/file.php?id=341&sid=bf7691cebded9118d6cc0f39fcdfbbbe)

**Reply by sethvoltz** [reply]
The hexagonal buttons are sick. This would also be cool as a button bar across the top of the CR-6 SE if I hadn’t already top-mounted my spool.

**Comment by Andy-ABTec** [comment]


**Reply by Andy-ABTec** [reply]
# Is this the smallest case ever?

![screenGrab](https://user-images.githubusercontent.com/5175695/107068285-531df080-67d8-11eb-8517-cf188f8f2565.png)

## This case design, including the Fusion Archive, can be found on [Thingiverse](https://www.thingiverse.com/thing:4750712)...

![103472785-7ee92900-4d89-11eb-8c55-fc4e82f8b98c](https://user-images.githubusercontent.com/5175695/106097967-214dbf80-6130-11eb-8f4f-345f2cbf0fd0.jpg)

First prototype, printed at 0.12mm, it was an experiment with "Z Hop" in Cura to see how well I could "insert" the text in a different colour by changing the filament and overprinting (white first, then black). I was pretty amazed when I took it off the bed.

The lack of tolerances on the fit was deliberate on my part, I wanted a friction fit that didn't keep falling apart. The detent in the base lets you apply an initial tweak with a knife to slide the case open.

One of the biggest problems with the whole design was finding push buttons small enough to fit in the available space.

The width and depth were specific to allow mounting across 40mm V rail or along 20mm V rail using with the printed "T nut" add-ons for M3 button screws. If it's mounted from the back the cabling can still exit the back and run up or down in one of the rails - which reminds me I need to sketch some snap-in covers for the slots...

**Reply by sethvoltz** [reply]
The turn around from hearing about the overprinting technique to seeing it in your prints was so fast. I haven’t done any reprints of my case yet but if I make any tweaks, I’ll be adding this into the mix.

**Reply by Andy-ABTec** [reply]
It's fiddly, It relies on the slicer supporting Z Hop and the ability to align the origins from the STL files and the design program (in this case Fusion 360) needs to include those origins in the first place...

**Reply by sethvoltz** [reply]
Hmm, good point. I’d have to redesign my lid to have a component origin that worked for that. Maybe I can do a realign origin to make that work.

**Reply by sethvoltz** [reply]
Here’s the technique we’re talking about for those that come here later:

[Smooth 2 Color Inlays using Cura](https://youtu.be/0Sla-vIsvh4)

**Reply by Andy-ABTec** [reply]
Or if you want to go to extremes...

[_MakeAnything's_ Multicolour Multipass in Cura ](https://www.youtube.com/watch?v=D7FiT5VqLSg)

**Reply by sethvoltz** [reply]
Bad link?

**Reply by Andy-ABTec** [reply]
Oops - fixed - trying to be too clever and break the video out of the YouTube page...

**Reply by Andy-ABTec** [reply]
Hi @sethvoltz Fixed this a while ago and checked when I added my Thingiverse link earlier, think the last two posts (three including this one) can go now...

Think the beta stuff is giving me grief - I'll fiddle with the post again later..

**Comment by patofoto** [comment]
I love my little Micro Panel. I only need the display and not interested in the buttons. At least not yet. Like keeping track of the progress at the printer since my computer is not close sometimes. Here are some pictures of a redesigned housing I found on Thingiverse with the addition of the housing for the display. I wanted my Pi to be PoE powered and also get the feed from the Pi Camera via HDMI. Like neat cabling. Here are a few pictures of it:

![IMG_2718](https://user-images.githubusercontent.com/37857628/109439565-d5c35580-79fc-11eb-8dd0-6bf72b3a875a.jpeg)


![IMG_2719](https://user-images.githubusercontent.com/37857628/109439572-e4117180-79fc-11eb-8326-d9154318357f.jpeg)


![IMG_2722](https://user-images.githubusercontent.com/37857628/109439577-ed9ad980-79fc-11eb-9c16-a3cf05c06f9d.jpeg)


![IMG_2723](https://user-images.githubusercontent.com/37857628/109439640-2f2b8480-79fd-11eb-830f-6e1768ef2765.jpeg)



Also housed the camera and attached it to the Z gantry. Not the optimal position for a camera but good enough to keep an eye on things. Happy with how it all came together.

![IMG_2726](https://user-images.githubusercontent.com/37857628/109439631-276be000-79fd-11eb-91d4-2d591fd24184.jpeg)

**Reply by Andy-ABTec** [reply]
Loving the camera - is it adjustable?

**Reply by patofoto** [reply]
@Andy-ABTec Thank you. Not this iteration. Working on a lower one to be able to see closer to the level of the print bed and have been toying with being able to adjust camera angle but not putting much effort into this since my camera ends up only pointing at same spot 99% of the time. Designed to be part of the current printer design and to keep an eye on operation of machine only. Not really interested in shooting time lapses or movies and the current position is not ideal for this as it sits a little too high from tip of print head. One design tradeoff to simplicity.

**Reply by Andy-ABTec** [reply]
I was just interested because I have a HeroMe Gen5 on my Ender 3 and am having a devil of a job "seeing" between the cooling duct and the bed...

**Reply by sethvoltz** [reply]
This build is fantastic. The yellow contrasting with the black case is very nice. Very clean

**Comment by NilsRo** [comment]
Thats my Anycubic Mega S version integrated in a tool holder.

![IMG_0539](https://user-images.githubusercontent.com/51311453/109807893-c3c4fb00-7c26-11eb-8850-23d6402da3d9.JPG)
